sopel/modules/units.py

```diff
@@ -36,6 +36,14 @@ def k_to_c(temp):
     return temp - 273.15
 
 
+def _extract_source(pattern, trigger) -> tuple[str, ...]:
```

Comment:
Shouldn't this use the `typing.Tuple` version?

Reply:
It can use either form, since this module has already done `from __future__ import annotations`. I couldn't tell if we had a defined opinion about that, but figured the 'modern' form is preferable since it does work and the type checker understands it.

Reply:
Hm, a non-exhaustive review of code search results for "tuple" shows that we use both. Sometimes, even, it's mixed in the same file… Think I'll retract this idea. A future typing pass can push the whole code base toward consistently using one or the other, maybe with the help of a flake8 (or other static analysis tool) plugin.